### PR TITLE
Added Polish translation

### PR DESCRIPTION
**Pull request by radrow**

### CODE REVIEW

File: src/js/locale/pl.yaml [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+  enabled: Odblokowane
+  show_disconnected_clients: Pokaż rozłączone klienty
+  no_connected_clients: Brak połączonych klientów
+  host: Host
+  port: Port
+  name: Nazwa
+  name_disconnected: Nazwa (rozłączono)
+  group: Grupa
+  new_group: '+ Nowa grupa'
+  latency: Latencja
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:195]: `Opóźnienie` sounds better.

File: src/js/locale/pl.yaml [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+  logging:
+    title: Logowanie
+    mopidy: Loguj Mopidy
+    pusher: Loguj Pusher
+    actions: Loguj Akcje
+    snapcast: Loguj Snapcast
+  create_notification: Stwórz powiadomienie
+  create_process: Stwórz powiadomienie procesu
+  run_test: Uruchom test procesu
+  call: Zawołaj funkcję
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:271]: `wywołaj`

File: src/js/locale/pl.yaml [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+  browse_directory:
+    title: Katalog
+  playlists:
+    title: Moje playlisty
+  tracks:
+    title: Moje utwory
+modal:
+  add_to_playlist:
+    title: Dodaj do playlisty
+    subtitle: 'Wybierz playlistę do dodania %{count} track%{plural} to'
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:461]: Last two words are not translated.

**Comment by MarcinWieczorek** [comment]
I've read the translation briefly and it looks alright. Thank you for contributing. Let's discuss my comments.

File: src/js/locale/pl.yaml (hunk 1 above)
Comment (radrow) [reply src/js/locale/pl.yaml:195]: In technical lanugage `Latencja` is imo more suitable

Comment (MarcinWieczorek) [reply src/js/locale/pl.yaml:195]: Which is a literal translation making no sense. Why would you assume the language needs to be technical? This software is for everyone.
> latencja «okres utajnienia choroby, popędu seksualnego u dzieci itp.»

File: src/js/locale/pl.yaml [hunk 4]
```diff
@@ -0,0 +1,532 @@
...
+# Common and reusable phrases
+##
+errors:
+  uri_not_found: 'Nie rozpoznano URI %{uri}'
+  could_not_load: 'Nie można załadować %{name}'
+  could_not_load_library: 'Nie można załadować %{name} od %{provider}'
+  no_results: Nic nie znaleziono
+  authorization_required: 'Musisz najpierw zautoryzować %{provider}'
+  enable_first: 'Odblokuj najpierw %{provider}'
+  no_service_worker: Nie wspierany service worker
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:13]: Probably should be `Niewspierany`.
Also I suppose there is no translation for `service worker`

File: src/js/locale/pl.yaml [hunk 5]
```diff
@@ -0,0 +1,532 @@
...
+  seconds: 
+    short: sek
+    singular: sekunda
+    plural: sekund
+  minutes: 
+    short: min
+    singular: minuta
+    plural: minut
+  hours: 
+    short: gdz
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:86]: According to Polish Wikipedia page:
> Godzina, skrót godz., oznaczenie h (ang. hour)

Use either `godz` or `h`, I prefer the latter.

File: src/js/locale/pl.yaml [hunk 6]
```diff
@@ -0,0 +1,532 @@
...
+  edit_playlist:
+    title: Edytuj playlistę
+    title_create: Stwórz playlistę
+    name_required: Nazwa jest wymagana
+    name: Nazwa
+    description: Opis
+    provider: Dostawca
+    image:
+      label: Okładka
+      description: Tylko JPEG, 256kB maks. Zostaw puste by pozostawić okładę bez zmian.
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:482]: typo `okładę`

File: src/js/locale/pl.yaml [hunk 7]
```diff
@@ -0,0 +1,532 @@
...
+    options:
+      label: Opcje
+      public: Publiczne
+      collaborative: Kolaboracja
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:486]: I suggest `Współpraca` as `Kolaboracja` is considered negative by some sources. Also I was looking for an adjective form but couldn't find one.

File: src/js/locale/pl.yaml (hunk 1 above)
Comment (radrow) [reply src/js/locale/pl.yaml:195]: https://zakamarkiaudio.pl/2013/01/latencja-i-sposoby-jej-minimalizowania.html
https://nagrywamy.com/2020/05/06/wylaczanie-kompensacji-latencji-w-reaperze-sposob-drugi-oczywisty/
https://muzyczny.pl/portal/latencja-czyli-co-to-jest-i-jak-sobie-z-tym-radzic/
And most importantly: https://pl.wiktionary.org/wiki/latencja

This term is actually being used very commonly in that meaning and context. Also, when I hear "latencja" I think of a more or less regular delay of a streamed signal which matches this case very well. The difference is not deal changing tho, but leaves less doubts what is it about (i.e, something different than lag of regular requests).

Comment (MarcinWieczorek) [reply src/js/locale/pl.yaml:195]: Fair enough.

File: src/js/locale/pl.yaml [hunk 8]
```diff
@@ -0,0 +1,532 @@
...
+      label: URI wykonawców w bibliotece
+      description: URI używane do organizacji wykonawców w bibliotece
+    album_uri:
+      label: URI albumów w bibliotece
+      description: URI używane do organizacji albumów w bibliotece
+    connections:
+      label: Połączenia
+    version:
+      label: Wersja
+      upgrade_available: 'Dostępna aktualizacjia (%{version})'
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:360]: typo

File: src/js/locale/pl.yaml [hunk 9]
```diff
@@ -0,0 +1,532 @@
...
+        description: Kod kraju ISO 3166-1 alpha-2 (np. 'PL')
+      locale:
+        title: Lokalizacja
+        description: Kod językowy ISO 639 oraz ISO 3166-1 alpha-2 połączone podkreślnikiem (np. 'pl_PL')
+    icecast:
+      enable:
+        label: Odblokowane
+        description: Strumieniuj audio do tej przeglądarki
+      location:
+        label: Lokacja
```
Comment (MarcinWieczorek) [inline src/js/locale/pl.yaml:419]: Debatable
https://sjp.pwn.pl/poradnia/haslo/lokacja-i-lokalizacja;12628.html

Comment (radrow) [reply src/js/locale/pl.yaml:419]: As I understand "lokalizacja" is more related to some place in terrain, while "lokacja" describes placement in possibly some abstract space. I would go for the latter, but I am not pretty much sure.

Comment (MarcinWieczorek) [reply src/js/locale/pl.yaml:419]: I'm not sure either. Up to you.

Comment (radrow) [reply src/js/locale/pl.yaml:419]: I'd leave it as it is. The difference is really blurry anyway

File: src/js/locale/pl.yaml (hunk 8 above)
Comment (MarcinWieczorek) [reply src/js/locale/pl.yaml:360]: I don't think this one was fixed @radrow

**Pull request merged**